notebooks/official/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -76,12 +76,11 @@
         "\n",
```

Comment:
<p>Change to: Vertex AI Pipelines: Evaluating batch prediction results from AutoML Tabular regression model</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>done</p>